MdeModulePkg/Library/BrotliCustomDecompressLib/BrotliDecompress.c

```diff
@@ -182,7 +182,7 @@ BrGetDecodedSizeOfBuf (
   )
 {
   UINT64  DecodedSize;
-  INTN    Index;
+  UINT8   Index; // MU_CHANGE - comparison size mismatch

```

Comment:
This is really right? If the `StartOffset` is 0, will this loop ever end?

Reply:
You are right correct.

The code started out using INT8 and comparing to UINT8, so the updated change has been to use INT16 vs UINT8.